packages/cli/src/commands/commerce/catalog/create.ts

```diff
@@ -0,0 +1,139 @@
+import {bold} from 'chalk';
+import type {
+  CreateCatalogConfigurationModel,
+  CatalogConfigurationModel,
+  CatalogFieldsMapping,
+  New,
+  PlatformClient,
+} from '@coveord/platform-client';
+import {Command, Flags} from '@oclif/core';
+import {
+  HasNecessaryCoveoPrivileges,
+  IsAuthenticated,
+  Preconditions,
+} from '../../../lib/decorators/preconditions';
+import {Trackable} from '../../../lib/decorators/preconditions/trackable';
+import {AuthenticatedClient} from '../../../lib/platform/authenticatedClient';
+
+export default class CatalogCreate extends Command {
+  public static description = `${bold.bgYellow(
+    '(alpha)'
+  )} Create a commerce catalog interactively`;
+
+  public static flags = {
+    // TODO: find an alternative to skip all the prompts without having too many flags.
```

Comment:
Consider this as a UI command, i.e. we don't expect nor will support machine-user for this one